service/controller/v5/resource/chart/update.go

```diff
@@ -51,7 +51,7 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange inte
 			return microerror.Mask(err)
 		}
 
-		err = r.helmClient.UpdateReleaseFromTarball(ctx, releaseName, tarballPath, helm.UpdateValueOverrides(values))
+		err = r.helmClient.UpdateReleaseFromTarball(ctx, releaseName, tarballPath, helm.UpdateValueOverrides(values), helm.UpgradeForce(true))
```

Comment:
Didn't found a better solution than force upgrade in order to recover failed charts.

This might potentially delete/recreate resource. I haven't tested this.